src/openzaak/components/catalogi/api/serializers/zaaktype.py

```diff
@@ -66,9 +65,10 @@ class Meta:
             "domein": {"source": "broncatalogus_domein", "required": True},
             "rsin": {"source": "broncatalogus_rsin", "required": True},
         }
+        gegevensgroep = "broncatalogus"
 
 
-class BronZaaktypeSerializer(ModelSerializer):
+class BronZaaktypeSerializer(GegevensGroepSerializer):
     class Meta:
         model = ZaakType
         fields = ("url", "identificatie", "omschrijving")
```

Comment:
I think the `fields` and `extra_kwargs` can both be removed, because the `GegevensGroepSerializer` derives these values from the definition of the gegevensgroep on the model
```suggestion
```